build.gradle

```diff
@@ -76,3 +79,7 @@ gradlePlugin {
         }
     }
 }
+
+publishPlugins.onlyIf {
+    project.version ==~ /\d+\.\d+\.\d+/
```

Comment:
would prefer to use `versionDetails().isCleanTag` instead of duplicating regexes.